platform/ui/src/components/Icon/getIcon.jsx

```diff
@@ -1,28 +1,32 @@
 import React from 'react';
 // Icons
-import active from './../../assets/icons/active.svg';
+import seriesActive from './../../assets/icons/series-active.svg';
+import seriesInactive from './../../assets/icons/series-inactive.svg';
```

Comment:
We don't have to on this pass, but we should use generic names that describe these, and not application specific names.